code/style.css

```diff
@@ -1,4 +1,181 @@
+*, *::before, *::after { box-sizing: border-box; }
+
 html {
-  background: #0025ff;
-  color: #fff;
+  box-sizing: border-box;
+  margin: 0;
+  height: 100%;
+} /* Best practise in default - bescause of the box scale inside the border instead of outside = more predictable*/
+
+body {
+  background: rgb(206, 192, 247);
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif
 }
+
+/**** header with logo & navigation-style ****/
+.logo {  /* flexboxparent*/
+  background-color: rgb(206, 192, 247);
+  display:flex; /* flexbox - parent */
+  flex-direction: row;
+  width: 100%;
+  height: 300px;
+}
+
+.logo img{
+  padding: 0;
+  width: 200px;
+}
+
+.logo h1{
+  width: 100%;
+  height: 200px;
+  padding: 80px 90px 150px 90px;
+  font-size: 65px;
+  color: black;
+}
+
+.menu { /* flexbox-parent*/
+  display: flex;
+  flex-direction:row;
+  justify-content:center; 
+  background-color: rgb(206, 192, 247);
+}
+
+nav a {
+  color: rgb(19, 19, 36);
+  text-decoration: none;
+  font-size: 18px;
+  text-transform: uppercase;
+  padding: 10px 35px 45px 35px;
+  color: rgb(106 95 88);
+}
+
+nav a:hover{
+  color: black
+}
+
+/**** Head news ****/
+
+.head-news-container{ /* flexbox parent */
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  flex-flow: row wrap;
+  align-items: center;
+  justify-content: center;
+  margin: 20px 80px 20px 80px;
+}
+
+.head-news{ /* flexbox child */
+  color: black;
+  padding: 15px;
+  width: 50%;
+}
+
+.head-news img{
+  width: 100%;
+}
+
+.head-news h2{
+  width: 100%;
+  font-size: 54px;
+}
+
+.head-news p {
+  width: 92%;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+/**** News boxes ****/
+
+.news-box-container{  /* flexboxparent*/
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  flex-flow: row wrap;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+
+}
+.news-box {  /* flexboxchild*/
+  padding: 15px;
+  width: 20%;
+}
+
+.news-box img {
+  width: 100%;
+}
+
+.box-text {
+  font-size: 18px;
+  color: black;
+}
+
+/**** Footer ****/
+footer {
+  background-color: rgb(106 95 88);
+  left: 0px;
+  right: 0px;
+  height: 200px;
+}
+
+/**** Responsive IPAD ****/
+@media screen and (max-width: 1024px) {
+  
+  .logo img{
+    width: 150px;
+  }
+  .logo h1{
+    height: 200px;
+    padding: 100px 80px 150px 90px;
+    font-size: 40px;
+  }
+
+  .head-news h2{
+    font-size: 44px;
+  }
+  
+  .news-box{
+    width: 50%;
+  } 
+}
+/**** Responsive MOBILE ****/
+@media screen and (max-width: 768px) {
+  
+  .logo h1{
+    display: none;
+  }
+
+  .logo{
+   align-items: center;
+   justify-content: center;
+
+  }
+
+  .menu{
+   flex-direction: column;
+   align-items: center;
+   justify-content: center;
+  }
```

Comment:
Really like that you made it into a column!! 💫
Just keep in mind that if you open this on your phone there'll a lot of scrolling so a tip is to make the space between the menu items smaller and maybe check the size of the logo. Minor details but it's good to know about these thing  and you've done so good!